### DISCUSSION TITLE
Ingest NIDM formatted datasets

### DISCUSSION THREAD

**Discussion by jdkent** [neurostuff/neurostore#1015] [Ideas]
neurovault/NiMARE support NIDM, so should neurostore

**Comment by adelavega** [comment]
I'm pretty hesitant to invest time into this at all given the scarcity, and that people sue NS-Compose to generate datasets. 

I think we can keep the issue but remove any milestone from this.

**Comment by jdkent** [comment]
converting to a discussion as an idea.
